compiler/ml/classify_function.ml

```diff
@@ -24,7 +24,7 @@
 
 let rec is_obj_literal (x : _ Flow_ast.Expression.t) : bool =
   match snd x with
-  | Identifier (_, {name = "undefined"}) | Literal _ -> true
+  | Identifier (_, {name = "undefined"}) | RegExpLiteral _ -> true
```

Comment:
Compile errors were fixed when I added the full list of `| StringLiteral _ | BooleanLiteral _ | ...` here like below.

Reply:
Ahah I missed this!